spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/support/DefaultRateLimitUtils.java

```diff
@@ -46,4 +50,9 @@ public String getRemoteAddress(final HttpServletRequest request) {
         }
         return request.getRemoteAddr();
     }
+
+    @Override
+    public Set<String> getUserRoles() {
+        throw new RuntimeException("Not supported");
```

Comment:
@lchayoun I've added it here but still not sure what to do, I was thinking either to add a more specific `Exception` or just return an `emptySet`. WDYT?